pages/linux/e2image.md

```diff
@@ -0,0 +1,24 @@
+# e2image
+
+> Save critical ext2/ext3/ext4 filesystem metadata to a file.
+> More information: <https://linux.die.net/man/8/e2image>.
+
+- Save metadata located on device to the specified file:
+
+`e2image {{/dev/sdXN}} {{path/to/image-file}}`
```

Comment:
The variable names should be in `snake_case`, so it should be `image_file` but I think just `file` is enough. What do you think?
```suggestion
`e2image {{/dev/sdXN}} {{path/to/file}}`
```